python/paddle/fluid/layers/nn.py

```diff
@@ -207,6 +207,7 @@
     'deformable_conv',
     'unfold',
     'deformable_roi_pooling',
+    'filter_instag',
```

Comment:
do you filter instance by ins tag of just filter instag? could you give a more meaningful name for this op?

Reply:
thank you it is filter_by_instag now